therr-client-web/src/styles/components/tile.scss

```diff
@@ -0,0 +1,87 @@
+.tile_wrapper {
```

Comment:
I just moved this file into components because I think we can reuse it for the analytics/metrics tiles